### DISCUSSION TITLE
How to view file history commit by commit in Lazy git?

### DISCUSSION THREAD

**Discussion by byteZorvin** [jesseduffield/lazygit#3320] [Q&A] [answered]
I have used git lens in VsCode and its file history feature. I was wondering if I could get the same somehow in my lazy git setup. The top right symbol let me go to the previous commit just for that particular file.
![Screenshot from 2024-02-11 17-44-36](https://github.com/jesseduffield/lazygit/assets/75808688/2d593f57-aeb5-4722-b6e8-4a8322127794)

**Comment by mark2185** [comment, marked as answer]
Go to the `Commits` panel and try pressing `ctrl+s`.
